internal/services/logz/logz_common.go

```diff
@@ -8,6 +8,7 @@ import (
 )
 
 const TagRuleName = "default"
+const PlanIdSuffix = "@TIDgmz7xq9ge3py"
```

Comment:
.. what's the context for this? Is this only applicable to Azure Public?

Reply:
**"@TIDgmz7xq9ge3py"** is the billing term id for the plan. Previously, there is only one id for all plans. However, Azure backend changed it from **one term id for all plans** to **each plan with a single term id** and also updated the enumeration value of **planDetails** from **"100gb14days"** to **"100gb14days@TIDgmz7xq9ge3py"**, which cause all testcases failed.

Reply:
is there a way to programatically get this? or is it just a magic value callers are expected to know?

Reply:
It's a magic value. Currently there is no public REST API to get the term ID. The service team is considering writing a document to describe how to use such magic value in the REST API.